### PR TITLE
(18) Council data download page

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Add links to data downloads for all the available local authorities
- Move the download link for the votes.tsv file into the new Data page
- Link the Data page from the FAQ

#### Data page
Screenshot 2022-04-22 at 14 26 40

#### FAQ page
Screenshot 2022-04-22 at 14 27 12

### CODE REVIEW

**Comment by CristinaRO** [comment]
[Council data download page](https://trello.com/c/M3jBgjEu/18-council-data-download-page)

File: db/schema.rb [hunk 1]
```diff
@@ -10,10 +10,16 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_04_14_131424) do
+ActiveRecord::Schema[7.0].define(version: 2022_04_21_162919) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "local_authorities", force: :cascade do |t|
```
Comment (pezholio) [inline db/schema.rb:17]: Given this data is pretty much static - I wonder if it's better to read from a config file?

Comment (CristinaRO) [reply db/schema.rb:17]: The thinking was that it might not be static, and being database backed would give us flexibility in the future.

Comment (pezholio) [reply db/schema.rb:17]: 👍

**Review by pezholio — APPROVED** [review]
LGTM 👍

**Pull request merged**